lib/stat_tracker.rb

```diff
@@ -493,4 +491,51 @@ def worst_defense
     team_id = all_teams_opponent_averages.key(all_teams_opponent_averages.values.max)
     get_team_name_from_id(team_id)
   end
+
+  def find_games_by_team_id(team_id, games = @games)
```

Comment:
I wrote this method. I wanted to use games_by_team_id(team_id), but that returns a game_team instance. I wanted to then use that game_team instance to find all the games but again, was getting to be long. 

Please let me know if you find a better alternative for this.